tests/api.test.js

```diff
@@ -2,7 +2,7 @@ import { jest } from "@jest/globals";
 import axios from "axios";
 import MockAdapter from "axios-mock-adapter";
 import api from "../api/index.js";
-import { calculateRank } from "../src/calculateRank.js";
+import { calculateRank } from "../src/fetchers/helpers/calculateRank.js";
```

Comment:
maybe calculate rank can be inside common folder, but adding another helper folder just for calculateRank file seems a bit overkill.

Reply:
Agreed I changed the structure. 👍